Content.Server/GameObjects/EntitySystems/DisassembleOnActivateSystem.cs

```diff
@@ -17,16 +21,23 @@ public override void Initialize()
             SubscribeLocalEvent<DisassembleOnActivateComponent, ActivateInWorldMessage>(HandleActivateInWorld);
         }
 
+        public override void Shutdown()
+        {
+            base.Shutdown();
+
+            UnsubscribeLocalEvent<DisassembleOnActivateComponent, ActivateInWorldMessage>(HandleActivateInWorld);
+        }
+
         private async void HandleActivateInWorld(EntityUid uid, DisassembleOnActivateComponent component, ActivateInWorldMessage args)
```

Comment:
The cancellation token should prevent this if you address the review below, but please make sure this can't be called twice or more at once